docs/advanced_topics/jinja2.rst

```diff
@@ -0,0 +1,97 @@
+.. _jinja2:
+
+=======================
+Jinja2 template support
+=======================
+
+Wagtail supports Jinja2 templating for all front end features. More information on each of the template tags below can be found in the :ref:`writing_templates` documentation.
+
+Configuring Django
+==================
+
+Django needs to be configured to support Jinja2 templates. As the Wagtail admin is written using regular Django templates, Django has to be configured to use both templating engines. Wagtail supports the Jinja2 backend that ships with Django 1.8 and above. Add the following configuration to the ``TEMPLATES`` setting for your app:
```

Comment:
Could we link to this doc? https://docs.djangoproject.com/en/1.8/topics/templates/#django.template.backends.jinja2.Jinja2

It includes some details on how to setup the environment which is required in order to use the ``url`` and ``static`` tags.